react/components/FilterOptionTemplate.js

```diff
@@ -152,7 +169,7 @@ const FilterOptionTemplate = ({
           <ShowMoreFilterButton
             quantity={quantity - FACETS_RENDER_THRESHOLD}
             truncated={truncated}
-            toggleTruncate={() => openTruncated(truncated => !truncated)}
+            toggleTruncate={() => openTruncated((isTruncated) => !isTruncated)}
```

Comment:
i think this will cause an error 🤔

Reply:
well, it should not, I just changed the function parameter name (it was a lint issue. the `no-shadow` one).

Reply:
ahhh sorry, for a moment I thought the function was using the same constant above. we see here a reason to use no-shadow hahah 😆

Reply:
hahahahah true! Now this rule makes sense